packs/data/feat-effects.db/effect-dragons-rage-wings.json

```
@@ -0,0 +1,48 @@
+{
+    "_id": "wmBSuZPqiDyUNwXH",
+    "img": "systems/pf2e/icons/spells/dragon-wings.webp",
+    "name": "Effect: Dragon's Rage Wings",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Dragon's Rage Wings]{Dragon's Rage Wings}</p>\n<p>While you are raging, you gain a fly Speed equal to your land Speed.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 12
+        },
+        "rules": [
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    "self:effect:rage"
+                ],
+                "selector": "fly-speed",
+                "value": "@actor.attributes.speed.value"
```

```suggestion
                "value": "@actor.attributes.speed.total"
```
`.value` is the pre modifier version, `.total` is after modifiers.